docs/ingestion/ingestion-spec.md

```diff
@@ -186,20 +186,28 @@ Treat `__time` as a millisecond timestamp: the number of milliseconds since Jan
 ### `dimensionsSpec`
 
 The `dimensionsSpec` is located in `dataSchema` → `dimensionsSpec` and is responsible for
-configuring [dimensions](./data-model.md#dimensions). An example `dimensionsSpec` is:
+configuring [dimensions](./data-model.md#dimensions). 
 
-```
+You can either manually specify the dimensions or take advantage of Schema auto-discovery where you allow Druid to infer all or some of the schema for your data. This means that you don't have to explicitly specify your dimensions and their type. 
+
+To use Schema auto-discovery, set `useSchemaDiscovery` to `true`. 
```

Comment:
I suppose we should continue to mention here that the classic string based mode can be used by leaving the dimensions list empty, or setting `includeAllDimension` but not `useSchemaDiscovery`.

`useSchemaDiscovery` = always use new 'auto' column indexers for any column that is discovery during ingestion
empty dimension list OR `includeAllDimensions` = classic string based schema discovery